Structure.Rmd

```diff
@@ -0,0 +1,398 @@
+
+To exclude a specific file or directory (the most common use case), you __MUST__ anchor the regular expression. For example, to exclude a directory called "notes", use `^notes$`. The regular expression `notes` will match any file name containing notes, e.g. `R/notes.R`, `man/important-notes.R`, `data/endnotes.Rdata`, etc. The safest way to exclude a specific file or directory is to use `usethis::use_build_ignore("notes")`, which does the escaping for you.
+
+`.Rbuildignore` is a way to resolve some of the tension between the practices that support your development process and CRAN's requirements for submission and distribution. The affected files fall into two broad, semi-overlapping classes:
+
+  * Files that help you generate package contents programmatically. Examples:
+    - Using `README.Rmd` to generate an informative and current `README.md`.
+    - Storing `.R` scripts to create and update internal or exported data.
+  * Files that drive package development, checking, and documentation, outside
+    of CRAN's purview. Examples:
+    - Files relating to the RStudio IDE.
+    - Using the [pkgdown package](https://pkgdown.r-lib.org) to generate a
+      website.
+    - Configuration files related to continuous integration/deployment and 
+      monitoring test coverage.
+
+Here is a non-exhaustive list of typical entries in the `.Rbuildignore` file for a package in the tidyverse:
+
+```
+^.*\.Rproj$         # Designates the directory as an RStudio Project
+^\.Rproj\.user$     # Used by RStudio for temporary files
+^README\.Rmd$       # An Rmd file used to generate README.md
+^LICENSE\.md$       # Full text of the license
+^cran-comments\.md$ # Comments for CRAN submission
+^\.travis\.yml$     # Used by Travis-CI for continuous integration testing
+^data-raw$          # Code used to create data included in the package
+^pkgdown$           # Resources used for the package website
+^_pkgdown\.yml$     # Configuration info for the package website
+^\.github$          # Contributing guidelines, CoC, issue templates, etc.
+```
+
+Note that the comments above are not generally present in `.Rbuildignore`; they are included here only for exposition.
+
+We'll mention when you need to add files to `.Rbuildignore` whenever it's important. Remember that `usethis::use_build_ignore()` is an attractive way to manage this file.
+
+### Binary package
+
+If you want to distribute your package to an R user who doesn't have package development tools, you'll need to make a __binary__ package. Like a package bundle, a binary package is a single file. Unlike a bundled package, a binary package is platform specific and there are two basic flavors: Windows and macOS. (Linux users are generally required to have the tools necessary to install from `.tar.gz` files.)
+
```

Comment:
Trying to word in such a way that doesn't deny the existence of prebuilt linux binaries, but also reflect what's currently mainstream practice.

Reply:
I think this needs a bit more to say that the group that builds most binaries is CRAN; generally, if you're making a public package, you'll rely on CRAN to build the binaries (and that's one of the most compelling reasons to get your package on CRAN).